tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -132,5 +132,6 @@ COPY install/.bazelrc /etc/bazel.bazelrc
 ENV TF_NEED_ROCM 1
 
 # This is a temporary workaround to fix Out-Of-Memory errors we are running into with XLA perf tests
-# By default, HIP runtime "hides" 200MB from the TF Runtime, we need to up that to 500MB
-ENV HIP_HIDDEN_FREE_MEM=500
+# By default, HIP runtime "hides" 256MB from the TF Runtime, but with recent changes (update to ROCm2.3, dynamic loading of roc* libs, et al)
```

Comment:
200, not 256?

Reply:
256 is the correct number

https://github.com/ROCm-Developer-Tools/HIP/blob/master/src/hip_hcc.cpp#L86

Reply:
got it. thanks for the clarification.